uk.co.compucorp.civicrm.hrcore/config/container/container.xml

```diff
@@ -45,6 +45,7 @@
       <argument id="civihr_stats_gatherer" type="service"/>
       <argument id="civihr.file_cache" type="service"/>
     </service>
+    <service id="custom_group.status_switcher" class="CRM_HRCore_Service_CustomGroupStatusSwitcher"/>
```

Comment:
This is something I've been thinking a bit about too, there's not really a convention here. Symfony best practices say we should use the [fully qualified class name](https://symfony.com/doc/current/contributing/code/standards.html#service-naming-conventions) for service IDs so if you're alright with that I can go ahead and start changing to that approach.

Reply:
This convention is somewhat recent (introduced in Symfony 3.4), because of the new autowiring feature in the Container which [requires services with FQCN in order to work](http://symfony.com/doc/current/service_container/autowiring.html#autowiring-logic-explained). For symfony 2.6 (the version used by CiviCRM), the [convention](https://symfony.com/doc/2.6/contributing/code/standards.html#service-naming-conventions) is similar to what I suggested. Let's follow the convention used for 2.6

Reply:
Yep, autowiring looks cool but you're right that it's a recent convention. I'll update the service IDs to match this style

Reply:
awesome! Thanks!